src/main/kotlin/uk/gov/dwp/dataworks/services/AuthenticationService.kt

```diff
@@ -56,16 +56,27 @@ class AuthenticationService {
                 .build()
                 .verify(userJwt)
 
-        return JWTObject(jwt, cognitoUsernameFromJwt(userJwt))
+        return JWTObject(
+                jwt,
+                userNameFromJwt(userJwt),
+                groupsFromJwt(userJwt)
+        )
     }
 
     /**
-     * Helper method to extract the Cognito username from a JWT Payload.
+     * Helper methods to extract the Cognito username and groups from a JWT Payload.
```

Comment:
This is bad practice for docs. Docs are assigned to each method individually and they do not necessarily show up next to each other in all places. For example if you do a "Quick Lookup" in IJ on `groupsFromJwt` you get no documentation about it. Even if the wording is virtually exactly the same we should be adding them to each function

Reply:
Sorry, I was not aware that that was how the docs work - I have remedied this